builtin/provisioners/chef/resource_provisioner.go

```diff
@@ -114,6 +115,9 @@ type provisioner struct {
 	UserKey               string
 	Vaults                map[string][]string
 	Version               string
+	RetryOnExitCode       []int
+	WaitForRetry          int
+	MaxRetries            int
```

Comment:
This list is ordered alphabetically to make it easy to search for certain fields. So can you please add these new ones in the correct order? Thanks!